doc/Overview/Mvux/ListStates.md

```diff
@@ -141,7 +136,7 @@ The `RemoveAllAsync` method uses a predicate to determine which items are to be
 
 ### ForEachAsync
 
-This operator can be called from an `IListState<T>` to execute an asynchronous action on all items currently in the list-state each time its data is changed (values are either added or removed):
+This operator can be called from an `IListState<T>` to execute an asynchronous action then the data changes. The action is invoked once for the entire set of data, rather than for individual items:
```

Comment:
```suggestion
This operator can be called from an `IListState<T>` to execute an asynchronous action when the data changes. The action is invoked once for the entire set of data, rather than for individual items:
```